### ISSUE TITLE
Update to conform with latest Elixir version.

### ISSUE THREAD

**Issue by shymega** [BennyHallett/elixir-rss#13]
When I compile my dependencies for my Elixir project that uses `elixir-rss`, I get this warning message:

```
==> rss
warning: the dependency :rss requires Elixir "~> 1.0.0" but you are running on v1.4.0
Compiling 1 file (.ex)
```

This would indicate to me that this library needs updating to conform with the latest Elixir version. As you can see, it seems to compile fine, so a fix for this would be as simple as updating mix.exs.

**Comment by shymega** [comment]
Just worth noting here, PR #12 seems to have been waiting for some time now, and _appears_ to address quite a few bugs. I would recommend merging that one in, and then updating the required version additionally afterwards.

**Comment by shymega** [comment]
Closing, fixed by #12.

**Comment by BennyHallett** [comment]
Thanks @shymega
